SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -179,9 +182,15 @@ void USpatialReceiver::LeaveCriticalSection()
 	PendingAuthorityChanges.Empty();
 }
 
-void USpatialReceiver::OnAddEntity(const Worker_AddEntityOp& Op)
+void USpatialReceiver::OnAddEntity(const Worker_Op& Op)
```

Comment:
As discussed offline - this code is definitely going away so if you want to trace this we need to put it in a connection handler. That said, do we need to trace add entity ops?

Reply:
Add entity ops are part of the spec on this ticket: https://improbableio.atlassian.net/browse/UNR-3964

Reply:
If this is needed then the code needs to live somewhere else. The ops don't really exist even at the moment.